validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/BlobsetSequences.java

```diff
@@ -99,7 +99,8 @@ public void endpoint_config_connection_success_reconnect() {
       return phase != null
           && phase.equals(BlobPhase.FINAL)
           && stateStatus.category.equals(SYSTEM_CONFIG_APPLY)
-          && stateStatus.level == Level.NOTICE.value();
+          && stateStatus.level == Level.NOTICE.value()
```

Comment:
I meant that the status in the blob block was as intended. The system status is not always indicative of what the blob is doing... the specific problem here was that the blob was reporting an error that was not checked by this condition -- this change doesn't fix that.

Reply:
Changed this to check blob state message for success.